app/src/main/java/com/github/se/polyfit/data/remote/firebase/PostFirebaseRepository.kt

```diff
@@ -159,6 +161,9 @@ class PostFirebaseRepository(
    * @param postCollection The Firestore collection to fetch the posts from.
    * @param completion The callback function to call once the posts have been fetched.
    */
+  private var previousNearbyKeys: List<String> = emptyList()
+  private var previousListOfPosts: List<Post> = emptyList()
+
```

Comment:
It seems to me that the value of `previousNearbyKeys` never changes throughout the function so I don't understand why we have an if statement `if(keys == preivousNearbyKeys)`

Reply:
I'm sorry you unfurtunatly review a outdated code for this part